### ISSUE TITLE
Failed to load class for data source: com.databricks.spark.xml

### ISSUE THREAD

**Issue by TYoung1221** [databricks/spark-xml#26]
Hello, I am using spark 1.4.0 with scala 2.10.4, when I try to use spark-xml this way, 

    bin/spark-shell --packages HyukjinKwon:spark-xml:0.1.1-s_2.10
    scala> import org.apache.spark.sql.SQLContext
    scala> val sqlContext = new SQLContext(sc)
    scala> val df = sqlContext.read
    .format("com.databricks.spark.xml")
    .option("rootTag", "book") 
    .load("books.xml")

Then I got this error,
    
    java.lang.RuntimeException: Failed to load class for data source: com.databricks.spark.xml

I read from https://issues.apache.org/jira/browse/SPARK-8368,  so do I need to move to spark 1.4.1 or 1.5, or could you please offer me some way to fix it?

**Comment by HyukjinKwon** [comment]
When it was in my private repository, I uploaded the package with the package name `org.apache.spark.*` as described [here](https://github.com/HyukjinKwon/spark-xml).
And the problem is, any release is not published and not uploaded yet since this package has moved to here Databricks. Namely, not you cannot use by `--packages` but have to manually build.
I will publish a release as soon as the pull requests I made are merged.

To cut it short, If you set the package name as described [here](https://github.com/HyukjinKwon/spark-xml), that should import properly.

I apologise that I made such a confusion. I think I should change the `README.md` at my private repository.
